sdk/cosmos/azure-cosmos/samples/nonpartitioned_container_operations.py

```diff
@@ -109,6 +109,7 @@ def create_nonpartitioned_container(db):
 
 
 def get_authorization(client, verb, resource_id_or_fullname, resource_type, headers):
+    # Here the method is being called
```

Comment:
This entire file looks like it should be... reconsidered...
There looks like too much manual url manipulation, direct request/response handling...
There must be a way to achieve this using standard API's...?

Reply:
Completely agreed, so much of this isn't even using the SDK directly... @kushagraThapar would love your input when you get a moment

Reply:
spoke with Kushagra and ended up removing a lot of the code - should be good now